tests/pfc_asym/conftest.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import time
+
+from helpers import Setup, OS_ROOT_DIR, ANSIBLE_ROOT
+
+
+PFC_GEN_FILE = "pfc_gen.py"
+PFC_FRAMES_NUMBER = 50000000
+PFC_QUEUE_INDEX = 0xff
+
+
+@pytest.fixture(scope="module")
+def ansible_facts(duthost):
+    """ Ansible facts fixture """
+    yield duthost.setup()['ansible_facts']
```

Comment:
Optional: We can use some parameters to limit the facts to be collected. For example:
```
yield duthost.setup(gather_subset="!all,!min,network", filter="ansible_Ethernet*")["ansible_facts"]
```

Reply:
Lets keep it as it is, to avoid possible issues